README.md

```diff
@@ -65,14 +65,18 @@ name: Differential ShellCheck
 on:
   push:
   pull_request:
-    branches: [main]
+    branches: [ main ]
```

Comment:
I think this could be `$default-branch`
https://github.blog/changelog/2020-07-22-github-actions-better-support-for-alternative-default-branch-names/

Reply:
That sounds great to me. Could you submit a PR?